net-p2p/retroshare/retroshare-0.6.5.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/RetroShare/RetroShare.git"
+EGIT_COMMIT="v${PV}"
+EGIT_SUBMODULES=()
+
+inherit desktop eapi7-ver git-r3 gnome2-utils qmake-utils versionator
+
+DESCRIPTION="P2P private sharing application"
+HOMEPAGE="https://retroshare.cc"
+
+# pegmarkdown can also be used with MIT
+LICENSE="AGPL-3 GPL-2 GPL-3 Apache-2.0 LGPL-2.1"
+SLOT="0"
+KEYWORDS=""
+
+IUSE="cli control-socket gnome-keyring +gui +jsonapi service +sqlcipher webui +xapian"
+
+REQUIRED_USE="
+	|| ( cli gui service )
+	service? ( jsonapi )"
+
+RDEPEND="
+	app-arch/bzip2
+	dev-libs/openssl:0=
+	>=dev-libs/rapidjson-1.1.0
+	net-libs/libupnp:0=
+	sys-libs/zlib
+	control-socket? ( dev-qt/qtnetwork:5 )
+	gnome-keyring? ( gnome-base/libgnome-keyring )
+	gui? (
+		dev-qt/qtcore:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtscript:5
+		dev-qt/qtxml:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtx11extras:5
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+	)
+	service? ( dev-qt/qtcore:5 )
+	sqlcipher? ( dev-db/sqlcipher )
+	!sqlcipher? ( dev-db/sqlite:3 )
+	webui? ( net-libs/libmicrohttpd )
+	xapian? ( dev-libs/xapian )"
+
+DEPEND="${RDEPEND}
+	gui? ( dev-qt/designer:5 )
+	jsonapi? (
+		app-doc/doxygen
+		dev-util/cmake
```

Comment:
they use `cmake` buildsystem for a feature of this package?

Reply:
yeah, retroshare team actually plan to move to cmake for the whole build system at some point, but now they depends on t only for some features